packages/pyright-internal/src/languageService/signatureHelpProvider.ts

```diff
@@ -141,13 +143,28 @@ export class SignatureHelpProvider {
                 paramName = params[params.length - 1].name || '';
             }
 
+            if (paramName && !paramString.includes(paramName) && paramName === 'kwargs') {
```

Comment:
Don't look for the name `kwargs`. That's a convention, but a `**` parameter can have any legal identifier name. Instead, use `functionType.details.parameters[paramIndex].category === ParameterCategory.VarArgDictionary`. Also, `paramString.includes(paramName)` is not a safe test because the name of the parameter could be found in one of the TypedDict field names.